net-analyzer/sdrplay/sdrplay-2.13.1-r1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit versionator unpacker udev multilib-minimal
+
+MY_PV_1=$(get_major_version)
+MY_PV_12=$(get_version_component_range 1-2)
+
+DESCRIPTION="SDRplay API/HW driver for all RSPs"
+HOMEPAGE="https://www.sdrplay.com"
+
+SRC_URI="
+	amd64? ( http://www.sdrplay.com/software/SDRplay_RSP_API-Linux-${PV}.run )
+	arm?   ( https://www.sdrplay.com/software/SDRplay_RSP_API-ARM32-${PV}.run  )
+	arm64? ( https://www.sdrplay.com/software/SDRplay_RSP_API-ARM64-${PV}.run )
+"
```

Comment:
This won't download anything for x86, which is in KEYWORDS below

Reply:
https://gitweb.gentoo.org/repo/gentoo.git/tree/net-wireless/sdrplay/sdrplay-2.13.1-r1.ebuild
copy paste , append... so flipping sleepy i'll drop keyword. 

 @ZeroChaos- @dwfreed  with me buiilding pentoo on arm64 like a machine till 3am over the last few weeks 
sleep has been at well a premium.